libs/src/services/schemaValidator/schemas/trackSchema.json

```diff
@@ -280,12 +280,12 @@
             "properties": {
                 "nft-collection": {
                     "type": "string"
+                },
+                "follow": {
+                    "type": "number"
```

Comment:
clarifying q: is the schema here `conditionName: value`? so the "number" value here is the user id of the user they have to follow? Might be worth having verbose names to keep that clear?

Reply:
yeah u got it right. yeah was thinking maybe follow-id?

Reply:
yeah or even `"follows_user_id"` idk, super nitty, whatever you think is best